src/main/java/core/hmhjdsd8/HlWorld9.java

```diff
@@ -0,0 +1,45 @@
+package core.hmhjdsd8;
+
+import core.model.Cat;
+import core.model.Dog;
+import java.time.LocalDate;
+
+public class HlWorld9 {
+    private String aa6sH8;
+    private int variable2;
```

Comment:
it's better to rename this variables to make code look more understandable and clean